apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -885,5 +945,186 @@ def _wait_for_qemu(self):
             raise ValueError(f"{item} not expected.")
 
 
+class ZephyrFvpMakeResult(enum.Enum):
+    FVP_STARTED = "fvp_started"
+    FVP_INIT = "fvp_initialized"
```

Comment:
how about we call this MICROTVM_API_SERVER_INIT?

Reply:
done